README.md

```diff
@@ -1,53 +1,19 @@
-In the result page, 
-for each result displaying card: utilising the data retrieved from the SEARCH input and display it into the specified div, together with details button leading to the DETAILS page, a favourite button to save the song to the favourites list, a PLAYING button which can play the song directly on the page, a YOUTUBE button which can lead to the external youtube page, and the MORE button which could display three more results
-In the details page, 
-for the DETAILS display, utilising the data retrieved from the SEARCH input and display more details such as the full-length lyrics, other tracks inside the albums, the released year, and other recommended singers/bands.
-for the GO BACK button, the user could go back to the result page
-Utilising APIs to retrieve data
-Combine all the files to complete the project
-
-
-The tasks of the project can be summarized as follows:
-One is responsible for the whole HTML and CSS layout for every page, and the SEARCH and FAVOURITES display in the index page. Also responsible for assembling all the functions to form a viable music application product.
-One is responsible for the RESULT displaying card on the RESULT page, utilising the data retrieved from API and display in the specified div, and the FAVOURITES button. The user should be able to add a song to the favourite list by clicking on the button. This should persist even if the user refreshes the page.
-One is responsible for the PLAYING button, YOUTUBE button on the RESULT page, and the MORE button. PLAYING is for playing the music on current page, YOUTUBE is for external link to YouTube music videos, and the MORE is for displaying three more other results. When there is less than three results left, the MORE button should disappear, and shows NO MORE RESULTS or some similar message instead. PLAYING button can be used by index page and details page. 
-One is responsible for the whole DETAILS page. Utilising the data from the SEARCH and displaying more detailed information, such as the Song’s name, singer/band’s name, albums with other tracks in the same album, the released year and displaying full-length lyrics. Also for the GO BACK button, which should allow the user to go back to the RESULT page. 
-
-Examples of existing application
-Shazam: https://www.shazam.com/track/498442693/dance-monkey-stripped-back
-Midomi: https://www.midomi.com/
-
-Our application is not to be exactly like the websites above but some of the layouts can be somehow alike, such as:
+## Getting Started
+Click [here](https://dmitriyaloyts.github.io/Music-Application/) for the live app.
 
+## Built With
 
+* HTML5
+* CSS3
+* [Bulma](https://bulma.io/documentation/)
+* JavaScript
+* [jQuery](https://api.jquery.com/)
+* RESTfull API's
```

Comment:
What is this REATfull API's meaning?

Reply:
It's 2 API's We used:

- https://api.audd.io
- https://theaudiodb.com